### ISSUE TITLE
[Question] How does triton inference server limit endpoint access for HTTP protocol ?

### ISSUE THREAD

**Issue by pradghos** [triton-inference-server/server#5837]
Currently, End point access can be restricted for GRPC Protocols 

ref: https://github.com/triton-inference-server/server/blob/r23.04/docs/customization_guide/inference_protocols.md

```python
--grpc-restricted-protocol=<protocol_1>,<protocol_2>,...:<restricted-key>=<restricted-value>

```
and one of the examples given - 

```python
tritonserver --grpc-restricted-protocol=shared-memory,model-config,model-repository,statistics,trace:<admin-key>=<admin-value> ...

```

Can we restrict endpoint access or achieve similar objective for HTTP protocol also ? Thank you !

**Comment by dyastremsky** [comment]
I don't believe so. The GRPC endpoint access restriction was recently added and is a beta feature.

CC: @GuanLuo

**Comment by GuanLuo** [comment]
That is correct, it is currently only implemented on GRPC frontend as beta feature. This can be a feature request for implementing the same on HTTP. @pradghos is the GRPC specification for this feature looks reasonable to you? Would like to collect more feedback on possible enhancement.

**Comment by pradghos** [comment]
@GuanLuo: Absolutely, it is reasonable and great feature to have in production environment. I look forward to see similar implementation for HTTP also. Any plan on which triton release this is going to be GA'ed (General availability) ? Thanks!

**Comment by dyastremsky** [comment]
Apologies for missing this earlier. I filed a ticket to investigate.

@GuanLuo can likely provide more details about when the GRPC version of this features shifts from beta to general availability. I think it's been a couple of releases now, so I imagine it may not be in beta anymore.

**Comment by yinggeh** [comment]
HTTP restricted API access feature was available here https://github.com/triton-inference-server/server/blob/main/docs/customization_guide/inference_protocols.md#httprest-and-grpc-protocols
